Makefile.am

```diff
@@ -63,6 +63,7 @@ install-data-local:: $(WEBPACK_INSTALL) $(MANIFESTS)
 install-data-local:: $(WEBPACK_GZ_INSTALL)
 	@for file in $^; do \
 		target="$(DESTDIR)$(pkgdatadir)/$${file##*dist/}.gz"; \
+		[ -n "$(DESTDIR)" -a "$${target}" -nt "$${file}" ] && continue; \
```

Comment:
I had to think about this a bit -- "newer" because gzip creates a new file, so if a file was not touched/rebuilt locally, the dest file.gz will generally be newer than the local file.

The commit message's subject is hard to read.

Reply:
I've reworked the subject line to be slightly less accurate but significantly easier to parse.

Sometimes staying under 50 characters has its cost...

Reply:
Btw: the fact that this adds a (serial) 1.85s to `make install` suggests to me that maybe we just want to do the compression as part of the webpack build... There are plugins for that...

That's another PR, for sure, though.